packages/api/src/learning-materials/learning-materials.controller.ts

```diff
@@ -21,7 +32,19 @@ export class LearningMaterialsController {
   }
 
   @Post()
-  createLearningMaterial(@JwtUserId() userId: UserId): Promise<CreateLearningMaterialResponse> {
-    return this.learningMaterialsService.createLearningMaterial(userId);
+  async createLearningMaterial(@JwtUserId() userId: UserId): Promise<CreateLearningMaterialResponse> {
+    try {
```

Comment:
I suppose that such code will repeat in many places, so i'd be nice to disgn API like this: 

this.learningMaterialsService.createLearningMaterial(userId)
.onError(ResourceNotFoundException, e => throw new NotFoundException(learningMaterialError.USER_NOT_FOUND))
.onError(...)

What do you think about that? 
Maybe another task to standardize it somehow.

Reply:
I thought about some error handling standardization as well, so yes, let's do it in a separate task